Objects/object.c

```diff
@@ -159,11 +159,8 @@ _PyDebug_PrintTotalRefs(void) {
    Do not call them otherwise, they do not initialize the object! */
 
 #ifdef Py_TRACE_REFS
-/* Head of circular doubly-linked list of all objects.  These are linked
```

Comment:
This comment should probably be preserved (and updated), not just removed.

Reply:
I moved the comment to the header file.